tests/benchdnn/conv/conv.cpp

```diff
@@ -667,6 +668,47 @@ void check_known_skipped_case(const prb_t *prb, res_t *res) {
     if (prb->alg == WINO) {
         if (is_cpu()) {
 #if DNNL_CPU_RUNTIME != DNNL_RUNTIME_NONE
+#if DNNL_AARCH64
+            bool pad_ok_f32 = prb->pw <= 1 && prb->ph <= 1 && prb->pw_r <= 1
+                    && prb->ph_r <= 1;
+            bool shape_ok = prb->ndims == 4 && prb->g == 1 && prb->kh == 3
+                    && prb->kw == 3 && prb->sh == 1 && prb->sw == 1
+                    && prb->dh == 0 && prb->dw == 0 && !prb->has_groups
+                    && pad_ok_f32;
+
+            const bool dir_ok = (prb->dir & FLAG_FWD);
+
+            const auto &po = prb->attr.post_ops;
+
+            // "true" here stands for eltwise.scale == 1.f check
+            const auto is_eltwise
+                    = [&](int idx) { return po.entry[idx].is_eltwise_kind(); };
+            auto is_sum = [&](int idx) { return po.entry[idx].is_sum_kind(); };
+
+            const bool sum_with_eltwise
+                    = (po.len() == 2) && is_sum(0) && is_eltwise(1);
+            const bool eltwise_only = (po.len() == 1) ? is_eltwise(0) : false;
+            bool eltwise_ok = false;
+
+            // Compute Library supports only one eltwise post-op or
+            // sum+eltwise post-ops
+            if (eltwise_only || sum_with_eltwise) {
+                const auto act_type = po.entry[sum_with_eltwise].eltwise.alg;
+                eltwise_ok
+                        = one_of(act_type, dnnl_eltwise_relu, dnnl_eltwise_tanh,
```

Comment:
Nit: please consider using `std::any_of` instead of introducing custom methods in conv specific files.

Reply:
Hi, it's not a custom method, it's an existing method found in `src/common/utils.hpp`, but I couldn't find any way of including that method without breaking that code. I am happy to implement with any_of, but I'm just wondering if it's possible to use `one_of` from `src/common/utils.hpp`

Reply:
Ignore me, I wasn't accessing the namespace correctly. I have changed this to use the existing one_of function.